easypy/resilience.py

```diff
@@ -241,19 +243,24 @@ def resilience(msg="ignoring error {type}", acceptable=Exception, unacceptable=(
     ...     print('after')
     before
     """
+    from .bunch import Bunch
+
     if unacceptable is None:
         unacceptable = ()
     elif isinstance(unacceptable, tuple):
         unacceptable += UNACCEPTABLE_EXCEPTIONS
     else:
         unacceptable = (unacceptable,) + UNACCEPTABLE_EXCEPTIONS
+
+    container = Bunch(exception=None)
```

Comment:
I think this justifies its own class. That - or we can introduce a `Cell` class to easypy.